### DISCUSSION TITLE
Saving a project with .cs extension

### DISCUSSION THREAD

**Discussion by Don1L** [thebaselab/codeapp#540] [Q&A]
I have a question, is it possible to save projects with the .cs extension in the full version of the program and open them on a computer in vs, just in the test version, when saving the project and opening it in vs, it is completely empty

**Comment by bummoblizard** [comment]
Sorry for not seeing this. Files should be able to save regardless its extension name. Make sure you saved the file by pressing command-s or the hide keyboard button.
